packages/utilities/psammead-styles/src/index.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import notes from '../README.md';
+import * as colours from './colours';
+
+const ColourContainer = styled.div`
+  padding: 10px;
```

Comment:
This all looks great locally. It'd be great to use our GEL spacings and Reith font for this, so that our Storybook example demonstrates usage of our utilities too.

Reply:
I added the `getBodyCopy(latin)`, `GEL_FF_REITH_SAN` and some GEL Spacing.